tfx/components/example_validator/component.py

```diff
@@ -91,10 +98,27 @@ def __init__(self,
           'been renamed to "statistics" and is deprecated. Please update your '
           'usage as support for this argument will be removed soon.')
       statistics = stats
-    anomalies = output or types.Channel(
-        type=standard_artifacts.ExampleAnomalies,
-        artifacts=[standard_artifacts.ExampleAnomalies()])
+    if exclude_splits is None:
+      exclude_splits = ['train']
```

Comment:
why is the train split always excluded from validation?

Reply:
because this is the current behavior -- before we added exclude_splits, example validator is hardcode to only use 'eval' split (in other words, 'train' split is excluded). you can see the 'current behavior' listed for each component in the design doc.